main/unbound/unbound.initd

```diff
@@ -22,36 +25,20 @@ depend() {
 	after auth-dns
 }
 
-checkconfig() {
-	checkpath --directory /var/run/unbound
-	UNBOUND_PIDFILE=$("${UNBOUND_CHECKCONF}" -o pidfile "${UNBOUND_CONFFILE}")
-	return $?
-}
-
 configtest() {
-	ebegin "Checking ${SVCNAME} configuration"
-	checkconfig
+	ebegin "Checking $RC_SVCNAME configuration"
+	pidfile=$("$UNBOUND_CHECKCONF" -o pidfile "$UNBOUND_CONFFILE")
```

Comment:
it is there because of the extra command - configtest.

I will move it & remove the extra command